src/main/java/org/folio/orders/utils/HelperUtils.java

```diff
@@ -492,42 +495,71 @@ public static int calculateInventoryItemsQuantity(CompositePoLine compPOL) {
    * @see #calculateInventoryItemsQuantity(CompositePoLine)
    */
   public static int calculateInventoryItemsQuantity(CompositePoLine compPOL, List<Location> locations) {
-    if (compPOL.getCheckinItems() != null && compPOL.getCheckinItems()) return 0;
+    return IntStreamEx.of(calculatePiecesQuantity(compPOL, locations, true).values()).sum();
+  }
+
+  /**
+   * Calculates pieces quantity for specified locations.
+   *
+   * @param compPOL composite PO Line
+   * @param locations list of locations to calculate quantity for
+   * @return quantity of items expected in the inventory for PO Line
+   * @see #calculateInventoryItemsQuantity(CompositePoLine)
+   */
+  public static Map<Piece.Format, Integer> calculatePiecesQuantity(CompositePoLine compPOL, List<Location> locations, boolean withItem) {
+    if (compPOL.getCheckinItems() != null && compPOL.getCheckinItems()) return Collections.emptyMap();
```

Comment:
Please use curly bracket notation here... especially given that it's used for several one-liners below.